hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -169,7 +170,27 @@ report_policy -verbose -skip_empty_summary_status -compat -output vcdc.rpt ALL
 file mkdir ../REPORT/
 
 foreach mod $modules {
-  report_policy -verbose -skip_empty_summary_status -compat -output ../REPORT/vcdc.$mod.rpt -module $mod {NEW TO_BE_FIXED DEFERRED}
+  # Find unique modules
```

Comment:
This is cool, thanks for adding this!

That basically means that we could expand the table that we are presenting [here](https://reports.opentitan.org/hw/top_earlgrey/cdc/latest/results.html) to make the overall dashboard more comprehensive... I can take a stab at this when I have some time (will probably do that when I start boostrapping the RDC, since that one requires a new parser as well).